db/write_batch.cc

```diff
@@ -1337,13 +1367,14 @@ Status WriteBatchInternal::InsertInto(
     const WriteBatch* batch, ColumnFamilyMemTables* memtables,
     FlushScheduler* flush_scheduler, bool ignore_missing_column_families,
     uint64_t log_number, DB* db, bool concurrent_memtable_writes,
-    SequenceNumber* last_seq_used, bool* has_valid_writes) {
```

Comment:
This was actually a bad naming here. The seq will actually to the next seq instead of the last one. Fixing it as part of this patch.